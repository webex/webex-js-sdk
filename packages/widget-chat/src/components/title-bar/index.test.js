import React from 'react';
import {createRenderer} from 'react-addons-test-utils';

import TitleBar from '.';

let displayName;
let component;

describe(`TitleBar component`, () => {
  beforeEach(() => {
    const renderer = createRenderer();
    displayName = `test@testing.net`;
    component = renderer.render(
      <TitleBar displayName={displayName} />
    );
  });

  it(`renders`, () => {
    expect(component.type).toBe(`div`);
  });

});
