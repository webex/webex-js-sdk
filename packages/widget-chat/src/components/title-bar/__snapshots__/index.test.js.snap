exports[`TitleBar component renders correctly 1`] = `
<div
  className="title-bar">
  <div
    className="avatar-container">
    <div
      className="avatar"
      style={undefined}>
      <span
        className={undefined}>
        T
      </span>
    </div>
  </div>
  <h1
    className="title">
    test@testing.net
  </h1>
</div>
`;
